pkg/backend/path_creds.go

```diff
@@ -68,23 +68,28 @@ func (b *backend) credsUpdateOperation(ctx context.Context, req *logical.Request
 
 	key := credKey(data.Get("name").(string))
 
-	code, ok := data.GetOk("code")
-	if !ok {
-		return logical.ErrorResponse("missing code"), nil
+	var tok *oauth2.Token
+
+	if code, ok := data.GetOk("code"); ok {
```

Comment:
You might want to throw an error if both `code` and `refresh_token` are specified, since this configuration doesn't make sense.

Reply:
Done